src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6314,28 +6315,33 @@ namespace SimAirServingZones {
 				CalcSysSizing( AirLoopNum ).InpDesCoolAirFlow = CalcSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 				FinalSysSizing( AirLoopNum ).InpDesCoolAirFlow = FinalSysSizing( AirLoopNum ).DesCoolVolFlow * FractionOfAutosize;
 			} else if ( SELECT_CASE_var == FlowPerCoolingCapacity ) {
-				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity ) {
+				if ( FinalSysSizing( AirLoopNum ).CoolingCapMethod == FractionOfAutosizedCoolingCapacity || FinalSysSizing( AirLoopNum ).ScaledCoolingCapacity == DataSizing::AutoSize ) {
```

Comment:
Scaled cooling capacity is not an autosizeable field. Regardless, if CoolingCapMethod == FractionOfAutosizedCoolingCapacity you will execute the code, regardless of the ScaledCoolingCapacity value.

    N23, \field Fraction of Autosized Cooling Design Capacity
      \type real
      \minimum 0.0
      \note Enter the fraction of auto-sized cooling design capacity. Required field when the cooling
      \note design capacity method field is FractionOfAutosizedCoolingCapacity.